### PR TITLE
feature/typeof-window

### PR DESCRIPTION
**Pull request by PaulieScanlon**
Hey, i've added `typeof window !== 'undefined'` to `index.js`, i think this will resolve any SSR framework errors but would you mind investigating?

Thanks in advance

Paul

### CODE REVIEW

**Comment by PaulieScanlon** [comment]
I've created an issue for this PR which can be seen here [issues/5](https://github.com/vasturiano/three-geojson-geometry/issues/5)

**Comment by vasturiano** [comment]
@PaulieScanlon thanks for your PR! I think this change makes sense.
However, I see a lot of unrelated code changes in the PR. Could you update it so that it includes only the relevant small section?

**Comment by PaulieScanlon** [comment]
@vasturiano Hey, ty for taking a lot. 

Blast! 

I think those other changes are my prettier formatter messing with us. Lemme see if i can disable that.

Were you ok with the inclusion of the standard GitHub node `.gitignore`?

**Comment by PaulieScanlon** [comment]
@vasturiano Hey, ok i think i've got it this time. Not 100% sure what your tabs or spaces settings were by i disabled my prettier extension. There's still a lot of lines changed but most are because everything's now indented by one because it's nested in the `if`

Lemme know what you think!

Ty!

**Comment by vasturiano** [comment]
@PaulieScanlon I still see many changes in the code. Would you mind making it so only the `typeof window !== 'undefined'` is added and nothing else?
Also, to keep it on topic, please don't include any other unrelated changes like the `.gitignore` file.

**Comment by PaulieScanlon** [comment]
@vasturiano hey! sure thing, Just pushed again. This has a minimal amount, which should make it easier to see what i've added.

Thanks for you time!

File: src/index.js [hunk 1]
```diff
@@ -3,16 +3,18 @@ import {
   Float32BufferAttribute
 } from 'three';
 
+import earcut from 'earcut';
+
+import interpolateLine from './interpolateLine';
+
+if(window !== 'undefined') {
```
Comment (vasturiano) [inline src/index.js:10]: This is not quite right. I think it should instead be:
```js
const THREE = (typeof window !== 'undefined') && window.THREE
  ? window.THREE
  ...
```

File: src/index.js [hunk 2]
```diff
@@ -3,16 +3,18 @@ import {
...
 const THREE = window.THREE
   ? window.THREE // Prefer consumption from global THREE, if exists
   : {
   BufferGeometry,
   Float32BufferAttribute
 };
 
-import earcut from 'earcut';
```
Comment (vasturiano) [inline src/index.js:13]: Please let's not change the order of the import statements, it's unrelated to this PR.

**Comment by vasturiano** [comment]
Thx @PaulieScanlon!

**Pull request merged**